### DISCUSSION TITLE
Parent categories link from Home to sub categories as card view with two column Layout without Products.

### DISCUSSION THREAD

**Discussion by devomman** [inspireui/support#5411] [Working on Features]
Hi Team (@minhcasi ), Please Add the UI of Fluxstore need From Parent categories link from home use as banner or categories to another page as sub categories as card view with two column Layout without Products then after click the that sub categories then show all the sub categories products.

Please add this, it will be very useful for separated the sub categories and products.

Thanks.

**Comment by trunghieuvn** [comment]
Hi @devomman cool idea, you can explain idea layout with wireframe? 
your contribution is welcome :D

**Reply by devomman** [reply]
Hi @trunghieuvn, Here it is https://imgur.com/a/RjJ4hBm
